lib/auth/apiserver.go

```diff
@@ -128,6 +128,7 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.POST("/:version/server/credentials", srv.withAuth(srv.generateServerKeys))
 
 	srv.POST("/:version/remoteclusters", srv.withAuth(srv.createRemoteCluster))
+	srv.PUT("/:version/remoteclusters", srv.withAuth(srv.updateRemoteCluster))
```

Comment:
Just realized; I don't actually see anywhere that this is called remotely.  Isn't `UpdateRemoteCluster` only ever called by the `AuthServer`?  Why is this being exposed as a remote API?

Reply:
Ahh, I totally cargo-culted this.
You're right, we don't need this publicly available.
Removed.